docs/product/reference/search/searchable-properties/session-replay.mdx

```diff
@@ -280,4 +280,10 @@ Username, which is typically a better label than the `user.id`.
 
 - **Type:** string
 
+### `viewed_by_id`
+
+Sentry user IDs who have seen this Replay. Searchable as a scalar or bracketed list.
+
+- **Type:** (unsigned) integer
```

Comment:
None of the other fields mention scalar or list, and they use `number` or UUID as types. We don't need to be so specific/different to describe this one.

What we could/should describe, here or somewhere, is how to get an id for a given user.

```suggestion
Sentry user IDs who have seen this Replay.
- **Type:** number
```

Reply:
Got it. How would an external user lookup ids?

Reply:
Looks like it id is also in the url when you click to view a user via `/settings/members/`, but it's not printed to the page.

Reply:
Ok.. would this be an acceptable way to get ids for now? I guess we could also make a PR to display user id on the members page